test/oqs_test_evp_pkey_params.c

```diff
@@ -0,0 +1,613 @@
+// SPDX-License-Identifier: Apache-2.0 AND MIT
+
+#include "test_common.h"
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <openssl/core_names.h>
+#include <openssl/crypto.h>
+#include <openssl/err.h>
+#include <openssl/evp.h>
+#include <openssl/provider.h>
+
+/** \brief List of hybrid signature algorithms. */
+const char *kHybridSignatureAlgorithms[] = {
```

Comment:
This is a static list apparently not generated by jinja2, i.e., falling out of sync the moment new algorithms get added. Please add to `oqs-template` (`generate.py`) logic.

Reply:
On second thought: Might it be possible to completely do away with this list and replace it by alg names lists dynamically retrieved from the provider (similar to how the tests are set up)?

Reply:
It still is not OK to hard-code the alg names: We don't do this anywhere else. Please consider using `jinja2` templating or get rid of this list.